extensions/hazelcast-client/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: "Hazelcast Client"
+metadata:
+  keywords:
+  - "hazelcast-client"
+  - "hazelcast"
+  - "cache"
+  - "distributed cache"
+  - "caching"
+  guide: "https://quarkus.io/guides/hazelcast-client"
```

Comment:
You need to add a `hazelcast-client.adoc` in https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc to avoid the broken link